app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -101,6 +100,28 @@ open class FenixApplication : Application() {
             ExperimentsMetrics.activeExperiment.set(branchName)
         }
 
+        // When the `fenix-etp-5651` experiment is active, set up ETP settings and GV policy.
+        // Note that this will not take effect the first time the application has launched,
+        // since there won't be enough time for the experiments library to get a list of experiments.
+        // It will take effect the second time the application is launched.
+        Experiments.withExperiment("fenix-etp-5651") { branchName ->
+            when (branchName) {
+                "control_strict" -> {
+                    settings().setUseStrictTrackingProtection()
+                    components.useCases.settingsUseCases.updateTrackingProtection(
+                        components.core.createTrackingProtectionPolicy()
```

Comment:
Would you mind moving this to its own file?  The code in FenixApplication could look something like `ExperimentsManager().initEtpExperiment(this)`, or similar.